src/with_clause_parser.c

```diff
@@ -113,6 +114,24 @@ ts_with_clauses_parse(const List *def_elems, const WithClauseDefinition *args, S
 	return results;
 }
 
+extern TSDLLEXPORT char *
+ts_with_clause_result_unparse_value(const WithClauseResult *result)
```

Comment:
Our code mostly uses "deparse", let's use it here as well to avoid confusion.

Reply:
Agree on that.
```suggestion
ts_with_clause_result_deparse_value(const WithClauseResult *result)
```